examples/main.tf

```diff
@@ -49,7 +49,7 @@ resource "instaclustr_cluster" "example" {
 
   bundle {
     bundle = "SPARK"
-    version = "apache-spark:2.3.2"
+    version = "2.3.2"
```

Comment:
[Question] - Pretty sure this was working before. Was there a breaking API change that requires this?

Reply:
No, this is so we don't encourage usages of the internal versions (we try to avoid that now)

But in the tests we had to change to use the internal versions because some tests check against the plan and because the API response from terraform-description endpoint returns the ic version, it detects diff and fails the tests

Reply:
Ah got it, thanks for confirming. I'm guessing that means terraform import won't work with internal versions then. Either way all good.